src/fetcher.ts

```diff
@@ -0,0 +1,21 @@
+import * as Heroku from '@heroku-cli/schema'
+import {APIClient} from '@heroku-cli/command'
+
+export async function addon(heroku: APIClient, addon: string, app?: string) {
```

Comment:
nit: why not call this `fetchAddon` so you don't have to `import {addon as fetchAddon}` ?

Reply:
I copy/pasted from [another cli plugin](https://github.com/heroku/heroku-cli-plugin-data-maintenance/blob/master/src/fetcher.ts) and followed the pattern. We have a version of this in all of our plugins now. It'd be awfully nice to get rid of the duplication next time we need a plugin.

Reply:
Done